addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1676,7 +1681,20 @@ var SnippetEditor = Widget.extend({
                 this.dragState.currentHeight = (maxRowEnd - 1) * (gridProp.rowSize + gridProp.rowGap) - gridProp.rowGap;
             }
         }
-    }
+    },
+    /**
+     * Called when the mouse is released outside the page iframe (e.g. the
+     * editor panel in Website). This is only useful in Chrome, where the 'stop'
+     * event of jQuery Draggable does not trigger a 'mouseup' event outside of
+     * the "preview" page iframe.
+     *
+     * @private
+     */
+    _onParentWindowMouseup() {
+        if (window.parent.document !== this.ownerDocument) {
```

Comment:
This `if` should not be inside the event handler, it should be used to set up or not the event handler.

In the end, I think this should not be fixed here at all: could we not fix this at the `SmoothScrollOnDrag` level? (If not fixing the lib, fixing this as close as possible to the call to `.draggable(` (thus also potentially fixing other usecases).